README.md

```diff
@@ -40,3 +40,6 @@ Check out which organizations are using and contributing to Carvel: [Adopter's l
 ### Integrating with ytt
 
 If you want to integrate `ytt` within your own tooling, review our [APIs](examples/integrating-with-ytt/apis.md).
+
+### Bazel
+`ytt` can be run with [Bazel](https://bazel.build/). See [rules_ytt](https://github.com/ekhabarov/rules_ytt) for details.
```

Comment:
I think this belongs to a doc (probably a new section needed here https://carvel.dev/ytt/docs/v0.47.x/) or a blog post, but not in the README.md file.